code/modules/spells/spell_types/shapeshift.dm

```diff
@@ -18,7 +18,11 @@
 		/mob/living/simple_animal/pet/dog/corgi,\
 		/mob/living/simple_animal/hostile/carp/ranged/chaos,\
 		/mob/living/simple_animal/bot/ed209,\
-		/mob/living/simple_animal/hostile/construct/armored)
+		/mob/living/simple_animal/hostile/construct/armored,\
+		/mob/living/simple_animal/pet/cat,\
+		/mob/living/simple_animal/butterfly,\
+		/mob/living/simple_animal/pet/gondola,\
+		/mob/living/simple_animal/crab)
```

Comment:
Basically all of these are just suicide for the wizard and the gondola shouldn't be so common on top of that

Reply:
They are basically meant for stealth

Reply:
Want stealth? Go mouse